integration-tests/contract-estimation-including-reveal-tests.spec.ts

```diff
@@ -0,0 +1,370 @@
+import { DEFAULT_FEE, MANAGER_LAMBDA, TezosToolkit, Protocols } from "@taquito/taquito";
+import { Contract } from "taquito/src/contract/contract";
+import { CONFIGS } from "./config";
+import { originate, originate2, transferImplicit2 } from "./data/lambda";
+import { ligoSample } from "./data/ligo-simple-contract";
+import { managerCode } from "./data/manager_code";
+
+CONFIGS().forEach(({ lib, setup, knownBaker, createAddress, protocol, rpc }) => {
+  const Tezos = lib;
+
+  const edonet = (protocol === Protocols.PtEdo2Zk) ? test : test.skip;
+  const florencenet = (protocol === Protocols.PsFLorena) ? test : test.skip;
+
+  describe(`Estimate scenario using: ${rpc}`, () => {
+    let LowAmountTez: TezosToolkit;
+    let contract: Contract;
+    const amt = 2000000 + DEFAULT_FEE.REVEAL;
+
+    beforeAll(async (done) => {
```

Comment:
Not the way I did it. Let's say that we do an origination operation, that a reveal is needed, and that we want to include the reveal fee in the estimation, I merged both fees (reveal and origination) in one Estimate object because I did not want to break the API... Open to suggestion